src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
```

Comment:
Please move values 20, 40, 50 to variables and give them descriptive names

Reply:
fix